src/garage/envs/multi_env_wrapper.py

```diff
@@ -52,22 +52,29 @@ class MultiEnvWrapper(gym.Wrapper):
             A list of objects implementing gym.Env.
         sample_strategy (function(int, int)):
             Sample strategy to be used when sampling a new task.
-        metaworld_mt (bool): Set if the envs that are passed Metaworld
-            environments from Multitask Benchmark Environments. Primarily
-            disables wrapper from augmenting observations with task one-hot
-            ids.
+        has_one_hots (bool): If all envs have a provide their own one-hot ids
```

Comment:
i still don't understand what's going on here. these seem to specify details of what's being passed to the wrapper, rather than what the wrapper should do.

if the wrapper needs to know something about something it is passed, it should introspect that.

do we really need 3 booleans to decide whether to concatenate the onehot or not???